internal/postgresql/postgresql.go

```diff
@@ -53,6 +53,10 @@ var (
 
 var log = slog.S()
 
+type PGManager interface {
```

Comment:
Can you please add a `go:generate` comment su we know how to generate the mocks? Like in https://github.com/sorintlab/stolon/blob/44f222c92afa0afd7b6cfd13f93e29a5e443619d/internal/store/store.go#L25

Reply:
Added 👍